README.md

```diff
@@ -1 +1,62 @@
-# TicTacToe
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Tic Tac Toe Project
+
+![screenshot](https://github.com/denisdiaconu/TicTacToe/blob/main/Screen%20Shot%202021-02-17%20at%2008.47.26.png)
+
+In this project, we recreate the Tic Tac Toe game.
+The game requires two participants and each one of them has to choose a symbol.
+In turn one, the first player needs to put his symbol in one of the positions.
+in turn two the second player needs to put his symbol in one of the free positions.
+The cycle goes on until there is a win or a tie.
```

Comment:
- [ ] Please remove grammar and spelling errors. You can use Grammarly for your help.